src/App.tsx

```diff
@@ -37,7 +53,12 @@ const App = (): JSX.Element => {
           Pokémon GO Raiders
         </Footer>
       </Layout>
+
+      </ThemeSwitcherProvider>
```

Comment:
Nit: I feel like the children of this element should be indented another level. Perhaps re-indent the whole file?

Reply:
reformatted in next commit